provisioning_templates/snippet/set_hostname.erb

```diff
@@ -0,0 +1,15 @@
+<%#
+name: set_hostname
+snippet: true
+model: ProvisioningTemplate
+kind: snippet
+-%>
+hostname <%= @hostname %>
```

Comment:
Would it be better to use os major/family foreman facts or rather something like `which hostnamectl`?